src/autoscaler/db/db.go

```diff
@@ -34,3 +34,8 @@ type HistoryDB interface {
 	CanScaleApp(appId string) (bool, error)
 	Close() error
 }
+
+type ScheduleDB interface {
```

Comment:
If the end is lost which could happen, we still need this DB to go under the Pruner to catch the case of multiple schedules for an App as well as Apps that have been removed.